src/lib/php/Dao/SysConfigDao.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/*
+ SPDX-FileCopyrightText: ©  Soham Banerjee <sohambanerjee4abc@hotmail.com>
+
+ SPDX-License-Identifier: GPL-2.0-only
+*/
+
+namespace Fossology\Lib\Dao;
+
+use Fossology\Lib\Db\DbManager;
+use Monolog\Logger;
+
+class SysConfigDao
+{
+  const TYPE_MAP = array(
+    '1' => "int",
+    '2' => "text",
+    '3' => "textarea",
+    '4' => "password",
+    '5' => "dropdown",
+    '6' => "boolean"
+  );
+  /** @var DbManager */
+  private $dbManager;
+  /** @var Logger */
+  private $logger;
+
+  function __construct(DbManager $dbManager, Logger $logger)
+  {
+    $this->dbManager = $dbManager;
+    $this->logger = $logger;
+  }
+
+  /**
+   * \brief Fetch configuration rows.
+   */
+  function getConfigData()
+  {
+    global $PG_CONN;
+    $sql = "select * from sysconfig order by group_name, group_order";
+    $result = pg_query($PG_CONN, $sql);
+    return (pg_fetch_all($result));
+  }
+
+  /**
+   * Get all customise information for admin
+   * @return array
+   */
+  public function getCustomiseData($data)
+  {
+    $finalVal = [];
+    foreach ($data as $row) {
+      $type = self::TYPE_MAP[$row['vartype']];
+      if ($type == "int") {
+        $value = intval($row['conf_value']);
+      } else {
```

Comment:
Keeping the return type consistent as string will help consumer of the endpoint by using single class to read the data.